packages/support/lib/fs.js

```diff
@@ -141,6 +145,39 @@ const fs = {
         walker.destroy();
       }
     });
+  },
+  /**
+   * Reads the closest `package.json` file from absolute path `dir`.
+   * @param {string} dir - Directory to search from
+   * @throws {TypeError} If `dir` is not a nonempty string or relative path
+   * @throws {Error} If there were problems finding or reading a `package.json` file
+   * @returns {object} A parsed `package.json`
+   */
+  readPackageJsonFrom (dir) {
```

Comment:
It would be nice to have an e2e test for the newly added methods

Reply:
I totally wrote tests for these but they must have gotten stashed or otherwise lost.  I'll dig them up and push